pkg/ansible/proxy/proxy.go

```diff
@@ -251,3 +267,27 @@ func Run(done chan error, o Options) error {
 	}()
 	return nil
 }
+
+func addWatchToController(owner metav1.OwnerReference, cMap *aoController.ControllerMap, resource *unstructured.Unstructured) error {
+	gv, err := schema.ParseGroupVersion(owner.APIVersion)
+	if err != nil {
+		return err
+	}
+	gvk := schema.GroupVersionKind{
+		Group:   gv.Group,
+		Version: gv.Version,
+		Kind:    owner.Kind,
+	}
+	c, watch, ok := cMap.Get(gvk)
+	if !ok {
+		return errors.New("failed to find controller in map")
+	}
+	// Add a watch to controller
+	if watch {
+		err = c.Watch(&source.Kind{Type: resource}, &handler.EnqueueRequestForOwner{})
```

Comment:
I think you need to tell the `handler.EnqueueRequestForOwner` which type of owner to look for.